lib/sauce-launch-service.js

```diff
@@ -17,6 +17,14 @@ class SauceLaunchService {
         config.host = 'localhost'
         config.port = this.sauceConnectOpts.port || 4445
 
+        const sauceConnectTunnelIdentifier = this.sauceConnectOpts.tunnelIdentifier
+
+        if (sauceConnectTunnelIdentifier) {
+            capabilities.forEach((capability) => {
+                capability.tunnelIdentifier = sauceConnectTunnelIdentifier
```

Comment:
Let's make sure we don't just overwrite it. Sometimes you might want to use a different tunnel. So how about
```js
capability.tunnelIdentifier = capability.tunnelIdentifier || sauceConnectTunnelIdentifier
```